### ISSUE TITLE
[TW-1458] task 2.4.0.beta3 cannot be build without GnuTLS installed

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1484]
_Juergen Daubert on 2014-11-13T16:32:14Z says:_ 

 [ 24%] Building CXX object src/commands/CMakeFiles/commands.dir/Command.cpp.o
In file included from /tmp/task/src/task-2.4.0.beta3/src/commands/Command.cpp:78:0:
/tmp/task/src/task-2.4.0.beta3/src/commands/CmdSync.h:42:105: error: 'TLSClient' has not been declared
   bool send (const std::string&, const std::string&, const std::string&, const std::string&, const enum TLSClient::trust_level, const Msg&, Msg&);
                                                                                                         ^
src/commands/CMakeFiles/commands.dir/build.make:54: recipe for target 'src/commands/CMakeFiles/commands.dir/Command.cpp.o' failed

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-11-13T16:32:14Z
Modified: 2014-11-14T16:20:25Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2014-11-14T14:51:40Z says:_ 

 This was already fixed in the 2.4.0 branch with [commit 5b4793e7509b471c65934423e936c3cab58e793e|https://git.tasktools.org/projects/TM/repos/task/commits/5b4793e7509b471c65934423e936c3cab58e793e] a few days ago; unfortunately this fix didn't make it into beta3 at the time.

If you want to build beta3 without GnuTLS, manually applying that patch will help.

**Comment by taskwarrior** [comment]
_Juergen Daubert on 2014-11-14T16:20:25Z says:_ 

 Thanks, that works for me.
